web_widget_map/static/src/js/web_widget_map.js

```diff
@@ -0,0 +1,85 @@
+odoo.define('web_widget_map', function (require) {
+    "use strict";
+
+    var widget_registry = require('web.widget_registry');
+    var Widget = require('web.Widget');
+
+    var MapWidget = Widget.extend({
+        template: 'Map',
+        events: {
+            'click #loadMap': '_initializeMap',
+        },
+        init: function () {
+            this._super.apply(this, arguments);
+            this.mapLayers = [new ol.layer.Tile({
+                source: new ol.source.OSM()
+            })];
+            this.centerMap = [0, 0];
+            this.markers = [];
+        },
+        _initializeMap: function () {
```

Comment:
I'm not sure about the design. 
you are calling _initializeMap and in the hr module, it makes an rpc call.
https://github.com/OCA/hr/pull/731/files#diff-4f79c3ed43c3d1ec84dd1097f991c34aR17

but if you have two modules that depends on this widget, I think that the call will be done for both model, and generate errors. 
I think that refactoring is needed. don't you think ?